devenv/dev-dashboards/datasource-elasticsearch/elasticsearch_compare.json

```diff
@@ -10530,14 +10526,21 @@
       "id": 121,
       "options": {
         "legend": {
+          "asTable": false,
           "calcs": [],
           "displayMode": "list",
+          "isVisible": false,
           "placement": "bottom"
         },
         "tooltip": {
           "mode": "single"
         }
       },
+      "panelSchema": [
```

Comment:
this probably is not correct since it is optional with [number, number], we should ignore it.